src/PersistenceStrategy/MariaDbAggregateStreamStrategy.php

```diff
@@ -78,7 +78,7 @@ public function prepareData(Iterator $streamEvents): array
         return $data;
     }
 
-    public function indexedColumns(): array
+    public function indexedMetadaColumns(): array
```

Comment:
typo: `indexedMetadaColumns`

Reply:
also as metadata is json, I would prefer `fields` instead of `columns`, since json has no columns.